tests/collisions/README.md

```diff
@@ -0,0 +1,97 @@
+
+__collisionsTest__ is a brute force hash analyzer
+which will measure a 64-bit hash algorithm's collision rate
+by generating billions of hashes,
+and comparing the result to an "ideal" target.
+
+The test requires a very large amount of memory.
+By default, it will generate 24 billion of 64-bit hashes,
+requiring __192 GB of RAM__ for their storage.
+The number of hashes can be modified using command `--nbh=`.
+be aware that testing the collision ratio of 64-bit hashes
+requires a very large amount of hashes (several billions) for meaningful measurements.
```

Comment:
Be aware that testing the collision ratio of 64-bit hashes requires a very large amount of hashes (several billion) for meaningful measurements.